guides/common/modules/proc_configuring-smart-proxy-server-with-default-ssl-certificates-to-generate-and-sign-puppet-certificates.adoc

```diff
@@ -79,5 +79,5 @@ This command creates the following files on the system where you configure {Smar
 +
```

Comment:
I can't comment on it above, but the generated files should be the same as in the custom ca procedure.